api/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.jpa.hibernate.naming_strategy=org.hibernate.cfg.EJB3NamingStrategy
```

Comment:
should change this to: `spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl`